examples/riscv/common/bir_symbLib.sml

```diff
@@ -0,0 +1,53 @@
+structure bir_symbLib :> bir_symbLib =
+struct
+
+open Abbrev;
+
+local 
+  open HolKernel Parse boolLib bossLib;
+  open bitTheory;
+  open birs_stepLib;
+  open birs_composeLib;
+  open birs_driveLib;
+  open birs_auxTheory;
+in
+
+fun bir_symb_analysis bprog_tm
+ birs_state_init_lbl birs_stop_lbls
+ bprog_envtyl birs_pcond =
```

Comment:
maybe we could remove the bprog_envtyl and simply compute it from the program "internally"?

Reply:
or do we need it outside currently?